mapping-for-navigation/_posts/0000-01-08-implicit-maneuvers.md

```diff
@@ -0,0 +1,71 @@
+# Implicit maneuvers
```

Comment:
@srividyacb You'll want to add a Jekyll header like the other posts. To do so, you will need to replace this first line with:

```
---
title: Implicit maneuvers
---
```